front/src/applications/rollingStockEditor/consts.ts

```diff
@@ -53,28 +66,157 @@ export enum Metadata {
   unit = 'unit',
 }
 
-export enum Parameter {
+export enum RollingStockEditorParameter {
   length = 'length',
   mass = 'mass',
-  max_speed = 'max_speed',
-  startup_time = 'startup_time',
-  startup_acceleration = 'startup_acceleration',
-  comfort_acceleration = 'comfort_acceleration',
-  gamma_value = 'gamma_value',
-  inertia_coefficient = 'inertia_coefficient',
-  loading_gauge = 'loading_gauge',
-  rolling_resistance_A = 'rolling_resistance_A',
-  rolling_resistance_B = 'rolling_resistance_B',
-  rolling_resistance_C = 'rolling_resistance_C',
-}
-
-export enum RollingStockResistance {
-  rolling_resistance_A = 'rolling_resistance_A',
-  rolling_resistance_B = 'rolling_resistance_B',
-  rolling_resistance_C = 'rolling_resistance_C',
+  maxSpeed = 'maxSpeed',
```

Comment:
Suggestion: no need for the repetition
```suggestion
  maxSpeed,
  ...
```

Reply:
Can't do that, otherwise the enum object is replaced by integers